src/cascadia/TerminalControl/ControlCore.h

```diff
@@ -341,6 +341,8 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
         til::point _contextMenuBufferPosition{ 0, 0 };
 
+        Windows::Foundation::Collections::IVector<int32_t> _cachedSearchResultRows{ winrt::single_threaded_vector<int32_t>() };

```

Comment:
This will heap allocate a new vector object for every terminal, even those that don't use search - is it important that we always have a real vector object in this slot? We're also going to destroy it immediately when somebody asks for it the first time.